extensions/blocks/revue/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * WordPress dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'revue';
+
+export const title = __( 'Revue', 'jetpack' );
```

Comment:
Does this need to be exported?

Reply:
Fixed in c608a48c37d89a1195f6fc280bc3ff21c6ca4400